src/HonzaBotner.Discord.Services/Commands/MemberCommands.cs

```diff
@@ -23,71 +23,159 @@ namespace HonzaBotner.Discord.Services.Commands
     [RequireMod]
     public class MemberCommands : BaseCommandModule
     {
-        private readonly HonzaBotnerDbContext _dbContext;
-        private readonly IHashService _hashService;
-        private readonly ILogger<MemberCommands> _logger;
-
-        public MemberCommands(HonzaBotnerDbContext dbContext, IHashService hashService, ILogger<MemberCommands> logger)
-        {
-            _dbContext = dbContext;
-            _hashService = hashService;
-            _logger = logger;
-        }
-
-        [Command("info")]
+        [Group("info")]
         [Aliases("about", "whois")]
         [Description("Provides info about a member.")]
-        [Priority(2)]
-        public async Task MemberInfo(CommandContext ctx,
-            [Description("Member to show info about.")]
-            DiscordMember member)
+        [ModuleLifespan(ModuleLifespan.Transient)]
+        public class MemberCommandsInfo : BaseCommandModule
         {
-            Verification? databaseRecord = await _dbContext.Verifications
-                .FirstOrDefaultAsync(v => v.UserId == member.Id);
-            await MemberInfoAsync(ctx, databaseRecord);
-        }
+            private readonly HonzaBotnerDbContext _dbContext;
```

Comment:
I know it was in previous implementation, but it is weird to have this direct access to database in what is basically controller.

Reply:
Yep, I wondered about that too. :D

Reply:
I started #174 for these enhancements. It is probably in multiple places in the codebase.